client/mysql.cc

```diff
@@ -3499,12 +3499,15 @@ com_help(String *buffer MY_ATTRIBUTE((unused)),
 	  return com_server_help(buffer,line,help_arg);
   }
 
-  put_info("\nFor information about Percona products and services, visit:\n"
+  my_snprintf((char*) glob_buffer.ptr(), glob_buffer.alloced_length(),
```

Comment:
Use c++-style casts (`xxx_cast<char*>()`) for the new code. I believe `const_cast` is enough here.